hardhat/hardhat.config.ts

```diff
@@ -34,6 +34,27 @@ const config: HardhatUserConfig = {
       url: `https://sepolia.infura.io/v3/${INFURA_KEY}`,
       accounts: [getAccount('admin').privateKey],
     },
+    arbitrumOne: {
+      chainId: 42161,
+      url: 'https://arb1.arbitrum.io/rpc',
+      accounts: [getAccount('admin').privateKey],
+    },
+    arbitrumNova: {
+      chainId: 42170,
+      url: 'https://nova.arbitrum.io/rpc',
+      accounts: [getAccount('admin').privateKey],
+    },
+    arbitrumSepolia: {
```

Comment:
Do we need the other configurations? Can we just keep this one?

Reply:
We've already added devnet and testnet in other PRs, so all these should be removed and the correct configs will be in place after rebasing.